cmd/root.go

```diff
@@ -66,38 +113,8 @@ func (a *MethodAws) InitRootCommand() {
 		Long:         "Audit AWS resources",
 		SilenceUsage: true,
 		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
-			var err error
-
-			awsConfig, err := awsconfig.LoadDefaultConfig(cmd.Context())
-			if err != nil {
-				return err
-			}
-			a.AwsConfig = &awsConfig
-
-			format, err := validateOutputFormat(outputFormat)
-			if err != nil {
-				return err
-			}
-			var outputFilePointer *string
-			if outputFile != "" {
-				outputFilePointer = &outputFile
-			} else {
-				outputFilePointer = nil
-			}
-			a.OutputConfig = writer.NewOutputConfig(outputFilePointer, format)
-
-			a.RootFlags.Regions, err = common.GetAWSRegions(cmd.Context(), *a.AwsConfig, a.RootFlags.Regions)
-			if err != nil || len(a.RootFlags.Regions) == 0 {
-				a.OutputSignal.Status = 401
-				a.OutputSignal.ErrorMessage = aws.String("No valid AWS regions found or specified")
-				return nil
-			}
-
-			cmd.SetContext(svc1log.WithLogger(cmd.Context(), config.InitializeLogging(cmd, &a.RootFlags)))
-
-			a.AwsConfig.Region = a.RootFlags.Regions[0]
-
-			return nil
+			authed := true
+			return a.setupCommonConfig(cmd, outputFormat, outputFile, authed)
```

Comment:
Nit. Just set `true` as the last parameter, no need to create a separate variable